libs/testing/e2e/src/lib/support/email-account.ts

```
@@ -0,0 +1,171 @@
+    subject: string | undefined
+    text: string | undefined
+    html: string | false
+  } | null>
+  email: string
+}) {
+  const client = new SESClient({ region: 'eu-west-1' })
+  await client.send(
+    new VerifyEmailAddressCommand({ EmailAddress: emailAccount.email }),
+  )
+  const verifyMsg = await emailAccount.getLastEmail(4)
+  if (verifyMsg && verifyMsg.text) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)